src/fontstash.h

```diff
@@ -886,14 +886,40 @@ static int fons__allocFont(FONScontext* stash)
 	return FONS_INVALID;
 }
 
+FILE* multibyte_open(const char*filename, const char*mode)
+{
+#ifdef _WIN32
+    int new_Len1 = 0;
```

Comment:
The variable names look inconsistent. How about `filenameLen`, `modeLen`, `wfilename`, `wmode`? `new_Len1&2` could just reuse a variable, i.e. just `len`.